tests/test_maya.py

```diff
@@ -32,6 +32,22 @@ def test_iso8601(string, expected):
     assert r == d.iso8601()
 
 
+@pytest.mark.parametrize("string,expected", [
+    ('January 1, 1970',
+     "12.17.16.7.5"),
```

Comment:
You can validate it easily using using WolframAlpha, for example: <https://www.wolframalpha.com/input/?i=January+1,+1970+to+maya+long+count>